src/angularComponent/ngcOmniboxSuggestionsDirective.js

```diff
@@ -1,14 +1,95 @@
 import NgcOmniboxSuggestionsController from './ngcOmniboxSuggestionsController.js';
 
-export default function ngcOmniboxSuggestionsDirective() {
+ngcOmniboxSuggestionsDirective.$inject = ['$document', '$templateCache'];
+export default function ngcOmniboxSuggestionsDirective($document, $templateCache) {
+
+  /**
+   * Takes in the public API template submitted by the app-maker and modifies it to repeat through
+   * the data structure of the Omnibox. The template will be different depending on whether it finds
+   * an ngc-omnibox-suggestion-category or not.
+   *
+   * Markup with an ngc-omnibox-suggestion-category:
+   *   <div ng-repeat="suggestion in suggestion.children || omnibox.suggestions">
+   *     <dl ngc-omnibox-suggestion-category ng-if="suggestion.children">
+   *       <dt>
+   *         <h5>{{suggestion.sample_category_title}}</h5>
+   *       </dt>
+   *       <dd>
```

Comment:
Hmm, that's tough since we're just modifying submitted markup vs creating much new markup of our own. Maybe I should document input -> output instead of just output.